python-avd/pyavd/_eos_designs/schema/schema_fragments/digital_twin.schema.yml

```
@@ -0,0 +1,91 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  digital_twin:
+    documentation_options:
+      table: digital-twin-configuration
+    type: dict
+    description: |-
+      PREVIEW: This option is marked as "preview", meaning the data models or generated configuration can change at any time.
+      Global settings to configure the Digital Twin of the Fabric.
+    default:
+      fabric: {}
```

Option 1

```suggestion
```

Option 2

```suggestion
default:
  environment: act
  fabric:
    act_username: admin
    act_password: admin
```